asreview/webapp/src/PreReviewComponents/ProjectPageOracle .js

```diff
@@ -0,0 +1,303 @@
+                  <Button
+                    className={classes.continuButton}
+                    variant={"outlined"}
+                    onClick={continueProjectSetup}
+                  >
+                    {state.info.projectHasDataset ? "Finish" : "Start"} setup
+                  </Button>
+                }
+
+                {(state.info.projectInitReady && !state.setup && !state.training) &&
+                  <Tooltip title="Download results">
+                    <IconButton
+                      aria-label="Export"
+                      onClick={props.toggleExportResult}
+                      color="inherit"
+                    >
+                      <GetAppIcon />
+                    </IconButton>
+                  </Tooltip>
+                }
+
+                {/* Project is ready, show button */}
+                {(state.info.projectInitReady && !state.setup && !state.training) &&
+                  <Button
+                    className={classes.continuButton}
+                    variant={"outlined"}
+                    onClick={()=>props.handleAppState("review")}
+                    disabled={state.finished}
+                  >
+                    Start reviewing
+                  </Button>
+                }
+
+                {(!state.info.projectInitReady && !state.setup && state.training) &&
+                  <div className={classes.wrapper}>
+                    <Button
+                      variant={"outlined"}
+                      disabled
+                      className={classes.continuButton}
+                      startIcon={<KeyboardVoiceIcon />}
+                    >
+                      Training model
+                    </Button>
+                    <CircularProgress size={24} className={classes.buttonProgress} />
+                  </div>
+
+                }
+              </Box>
+
+              {/* Project is not ready, continue setup */}
+              {state.training &&
+                <StartReview
+                  onReady={finishProjectFirstTraining}
+                />
+              }
+            </Grid>
+          </Grid>
+
+          {/* Cards on the project board */}
+          {!state.setup &&
+            <Box className={classes.cardBox}>
+              <StatisticsZone
+                project_id={props.project_id}
+                projectInitReady={state.info.projectInitReady}
+                training={state.training}
+              />
+              <PublicationZone
+                project_id={props.project_id}
+                showExportResult={state.info.projectInitReady && !state.setup && !state.training}
+                toggleExportResult={props.toggleExportResult}
+                reviewFinished={state.finished}
+                finishProject={finishProject}
+              />
+              <DangerZone
+                project_id={props.project_id}
+                handleAppState={props.handleAppState}
+              />
+            </Box>
```

Comment:
This should also be available on the project page for simulation. They have to see the statistics and they should be able to export the project as well as the results.

Reply:
One question: is finishing a project needed in case of a simulation? And what would it mean?

Reply:
I think a project is finished if the results are known... there is nothing more you can do once the simulation results are in. Maybe in the future we can add post-processing options of the data (for other graphics), but not in this stage and maybe I would prefer to do post-processing via R or Python scripts and not within ASReview.

Reply:
Oke, in that case I will remove the option to finish the project for a simulation.